src/helpers/issue.ts

```diff
@@ -657,9 +682,115 @@ export const getAvailableOpenedPullRequests = async (username: string) => {
       if (approvedReviews) result.push(pr);
     }
 
-    if (reviews.length === 0 && (new Date().getTime() - new Date(pr.created_at).getTime()) / (1000 * 60 * 60) >= botConfig.unassign.timeRangeForMaxIssue) {
+    if (reviews.length === 0 && (new Date().getTime() - new Date(pr.created_at).getTime()) / (1000 * 60 * 60) >= timeRangeForMaxIssue) {
       result.push(pr);
     }
   }
   return result;
 };
+
+// Strips out all links from the body of an issue or pull request and fetches the conversational context from each linked issue or pull request
+export const getAllLinkedIssuesAndPullsInBody = async (issueNumber: number) => {
```

Comment:
I think we have a redundant function, something like `getAllLinkedPulls` I saw in the beginning of this review

Reply:
I'm not sure what's the context of this function, but this functions retrieves all URLs to lead to PRs and issues in the body of the issue which is different from `gitLinkedPrParser` which retrieves only linked PRs on the specific issue.